src/payment/google-verification.service.ts

```
@@ -0,0 +1,262 @@
+      this.logger.debug('Executing Python verification script');
+      const { stdout, stderr } = await execFileAsync(
+        this.pythonPath,
+        [this.scriptPath, packageName, productId, purchaseToken],
+        {
+          timeout: 30000, // 30 second timeout
+          encoding: 'utf8',
+        },
+      );
+
+      if (stderr) {
+        this.logger.warn(`Python script stderr received (len=${stderr.length})`);
```

ESLint

[error] Replace ``Python·script·stderr·received·(len=${stderr.length})`` with `⏎··········`Python·script·stderr·received·(len=${stderr.length})`,⏎········`

(prettier/prettier)